src/liballoc/collections/vec_deque.rs

```diff
@@ -2709,7 +2709,7 @@ impl<T: fmt::Debug> fmt::Debug for VecDeque<T> {
 
 #[stable(feature = "vecdeque_vec_conversions", since = "1.10.0")]
 impl<T> From<Vec<T>> for VecDeque<T> {
-    /// Turn a [`Vec<T>`] into a [`VecDeque<T>`].
+    /// Turn a `Vec<T>` into a `VecDeque<T>`.
```

Comment:
It should work if you add this at the end of the comment:
```
///
[`Vec<T>`]: ../../vec/struct.Vec.html
[`VecDeque<T>`]: struct.VecDeque.html
```
It's possible to verify it locally by opening `build/x86_64-unknown-linux-gnu/doc/alloc/collections/vec_deque/struct.VecDeque.html` in a web browser.

Reply:
@mati865 That's essentially what I originally tried in https://github.com/rust-lang/rust/pull/61447 (https://github.com/rust-lang/rust/commit/5c3689fd7e0faaad842234039ae5acdca20cb29c), but it doesn't work because these docs end up showing up at multiple levels:
- https://doc.rust-lang.org/nightly/std/collections/struct.VecDeque.html#impl-From%3CVecDeque%3CT%3E%3E and
- https://doc.rust-lang.org/nightly/std/collections/vec_deque/struct.VecDeque.html#impl-From%3CVecDeque%3CT%3E%3E

Reply:
Right, it won't work that way.

Reply:
To have links you need to refer to items by the usual rust paths, so `

```
[`Vec<T>`]: crate::vec::Vec
```

or something like that. These links can also refer relatively to anything that’s in scope so 

```
[`Vec<T>`](Vec)
```

should “just” work too.

Reply:
Cool, with `crate` I finally got this to work.